src/main/java/io/appium/java_client/clipboard/HasClipboard.java

```diff
@@ -72,7 +72,7 @@ default void setClipboardText(String text) {
      */
     default String getClipboardText() {
         byte[] base64decodedBytes = Base64
-                .getDecoder()
+                .getMimeDecoder()
```

Comment:
Did suggested this in last PR. How did i missed it. Good catch @TikhomirovSergey

Reply:
@mykola-mokhnach Last time when i locally built Appium's WDA(Forked) and tested clipboard of URL's where passing in iOS after your fix [here](https://github.com/appium/WebDriverAgent/pull/70) but now i upgraded my XCUI driver to latest [release](https://github.com/appium/appium-xcuitest-driver/blob/master/package.json#L11) but URL's stopped working after that. 

URL's are working only if i use mime decoding for URL.

Reply:
Can you make mime decoding for getClipboardUrl too? Else it is failing...